src/commands/dev/stop.ts

```diff
@@ -79,11 +84,19 @@ export default class DevStop extends BaseCommand {
       socket.write('stop', () => {
         socket.end();
       });
+
+      socket.on('error', (e: any) => {
+        if (e.code === 'ECONNREFUSED') {
+          socket.end();
+          this.runComposeStop(name);
+        } else {
```

Comment:
This is the fallback where if you have a socket file but the dev process is closed and the socket is no longer being used, we run compose stop.